AK/StringBase.h

```diff
@@ -0,0 +1,126 @@
+/*
+ * Copyright (c) 2023, Dan Klishch <danilklishch@gmail.com>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#pragma once
+
+#include <AK/Endian.h>
+#include <AK/Forward.h>
+
+namespace AK::Detail {
+
+class StringData;
+
+inline static constexpr size_t max_short_string_byte_count = sizeof(StringData*) - 1;
```

Comment:
SCREAMING_CASE for constants

Reply:
Huh? https://github.com/SerenityOS/serenity/pull/15796#discussion_r1004835681

Reply:
I can't speak for that comment (other than that it was a year ago), but:

https://github.com/SerenityOS/serenity/blob/master/Documentation/CodingStyle.md#names
> Use SCREAMING_CASE for constants (both global and static member variables)

Reply:
*softly reeeeeeeeeees in the distance*